templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl

```diff
@@ -48,7 +48,11 @@
 				{fbvFormSection title="manager.navigationMenus.form.path" for="path" required="true"}
 					{fbvElement type="text" id="path" value=$path required="true"}
 					<p>
-						{url|replace:"REPLACEME":"%PATH%"|assign:"exampleUrl" router=$smarty.const.ROUTE_PAGE context=$currentContext->getPath() page="navigationMenu" op="view" path="REPLACEME"}
+                        {if $currentContext}
```

Comment:
Space-based indents. Please config your editor to use tabs, and search/replace for these.